tpgtools/api/pubsub/subscription.yaml

```diff
@@ -0,0 +1,192 @@
+components:
```

Comment:
Adding this will make the magician overwrite the existing pubsub topic resource with a DCL-based one. I don't think that's what we want at the moment?

Reply:
Can we add one of these for eventarc trigger instead?

Reply:
That's true, that was more or less why I was keeping this as a separate branch. But I can switch it over to eventarc